model/src/channel/thread/auto_archive_duration.rs

```diff
@@ -11,7 +11,7 @@ pub enum AutoArchiveDuration {
     Day,
     ThreeDays,
     Week,
-    Unknown { value: u16 },
+    Other { value: u16 },
```

Comment:
I'm not sure I understand why this one is Other and not Unknown? I'm not aware of applications being able to set custom values for this, so this seems the same as other cases where we just haven't registered a value in the library.

Reply:
The difference here is that unlike the other "unknown" types, we know exactly what the value represents. IMO this enum shouldn't even exist because it only represents some UI buttons that are not present or defined in the api docs at all